libfwupd/fwupd-remote.c

```diff
@@ -153,6 +153,8 @@ fwupd_remote_get_suffix_for_keyring_kind (FwupdKeyringKind keyring_kind)
 		return ".asc";
 	if (keyring_kind == FWUPD_KEYRING_KIND_PKCS7)
 		return ".p7b";
+	if (keyring_kind == FWUPD_KEYRING_KIND_JCAT)
+		return ".jcat";
 	return NULL;
 }
```

Comment:
This unction isn't used anymore.  Maybe deprecate it now?

Reply:
Ahh, it's static. Nuked!